packages/compass-components/src/components/document-list/document-edit-actions-footer.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { useCallback, useEffect, useRef, useState } from 'react';
+import type { MongoServerError } from 'mongodb';
```

Comment:
compass-components should not depend directly on mongodb-specific stuff if possible (see README for this package), please let's abstract component in a way that donesn't require us to add mongodb driver as a dependency here